src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -162,7 +162,16 @@ static Vec3f D_80B6506C = { 0.0f, 3.0f, 0.0f };
 
 // bytes per limb, used in draw func? bit over my head
```

Comment:
```suggestion
```
Looks like the `sLimbIndexToBodyPartIndex` map (not sure we have a standardized name for this. (Should probably also rename the temp variable down below)

Reply:
Exactly, I avoided naming these because of how many different names there are for this, but I think it should be standardized. Current general names for this are:
`sLimbIndexToBodyPartsIndex`
`sLimbIndexToBodyPartsPosIndex`
`sLimbIndexToLimbPosIndex`
`sBodyParts`
`sBodyPartPosIndices`
`limbPosIndex`

And some more specific names given are:
`sLimbIndexToShadowBodyPartsIndex`
`gEnHyBodyParts`
`sGyorgSphElementIndices`

I can make this part of my cleanup, any preference?

Reply:
I'd be fine with that, though we should probably have the discussion in discord for more visibility.

Reply:
Though it might also be better to just have a separate PR that does them all at once, especially since it might involve more than just naming. For example we may want to add enums for the body parts. I also know petrie worked on using preprocessor magic to define the map array based on just the limbs.

Reply:
I went with `sLimbToBodyParts` but there's more to do with this that I'll leave to a separate PR